clearwater-infrastructure/PyZMQ/Makefile

```diff
@@ -16,7 +16,7 @@ endif
 
 bin/buildout: ${ENV_DIR}/bin/python
 	mkdir -p .buildout_downloads/dist
-	${ENV_DIR}/bin/easy_install "setuptools>0.7"
+	${ENV_DIR}/bin/easy_install "setuptools==18.0.1"
```

Comment:
We pin setuptools 24 elsewhere - https://github.com/Metaswitch/crest/blob/ea3e04f08a5b207985b3f01d5387a32ab39dcd85/Makefile#L69. Worth being consistent?